src/main/java/com/cleanroommc/groovyscript/compat/mods/prodigytech/ExplosionFurnace.java

```diff
@@ -97,6 +97,7 @@ public void validate(GroovyLog.Msg msg) {
             validateItems(msg, 1, 2, 1, 1);
             validateFluids(msg);
             msg.add(craftPerReagent <= 0, "craftPerReagent should be positive!");
+            msg.add(power <= 0, "power should be positive!");
```

Comment:
message is "power should be positive", actual check is error if "<= 0". since `0` is positive, the message is incorrect.

Reply:
![Screenshot_2024-07-04-10-18-59-499_com.android.chrome-edit.jpg](https://github.com/CleanroomMC/GroovyScript/assets/52964431/9d5e09c6-d1ad-4964-a657-8e0d8adb4bbe)

Reply:
in mathematics yes, in number representation in programming there is a positive and negative zero (`+0`, `-0`). i think you should specify that it must be greater than 0.
https://en.wikipedia.org/wiki/Signed_zero